docs/user-guide/data/search.md

```diff
@@ -17,19 +16,16 @@ When clicking on the search bar, the search history panel is displayed. The last
 
 ### Search rules
 
-* A search value must have a minimum length of **3** characters. If your search term has less than three characters (e.g. `is`), please add two asterisks (`*is*`).
```

Comment:
I activated a rule that the unordered list markers must be consistent within each document